sw/device/tests/rv_core_ibex_nmi_irq_test.c

```diff
@@ -107,52 +93,59 @@ static void alert_handler_config(void) {
 
   alert_handler_testutils_configure_all(&alert_handler, config,
                                         /*lock=*/kDifToggleDisabled);
-  // Enables alert handler irq.
-  CHECK_DIF_OK(dif_alert_handler_irq_set_enabled(
-      &alert_handler, kDifAlertHandlerIrqClassa, kDifToggleEnabled));
 }
 
 /**
- * Handle the wdog bark nmi irq.
+ * Handle the wdog bark NMI irq.
```

Comment:
That's something that is not clear for me, out of curiosity, what is the rule on using Captals for Acronimous? Why not IRQ as well?

Reply:
Ah good point. I just have a preference for capitals for acronyms, but did not correct all of them here, hehe. I can align it to make it more consistent.